src/components/Explorer/DropdownMenu.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div :class="$style.dropdown">
+    <button :class="$style.button" @click="open">
+      <div :class="$style.dropdown_trigger">
+        <p :class="$style.title">{{ title }}</p>
+        <icon name="chevron-down" :class="$style.icon" />
+      </div>
+    </button>
+    <div v-if="isOpen" :class="$style.dropdown_menu">
+      <p
+        v-for="(content, index) in contents"
+        :key="index"
+        :class="$style.contents"
+      >
+        {{ content.str }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { reactive } from 'vue'
```

Comment:
```suggestion
import { reactive, PropType } from 'vue'
import { DropdownOption } from './use/useOption'
```
後ろで使うので`PropType`と`DropdownOption`をimportしといてください
`DropdownOption`については後ろで定義